src/routes/goosepage/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Goosepage from './Goosepage';
+
+const title = 'The Goose Page';
+
+export default {
+
+    path: '/goosepage',
+
+        action() {
```

Comment:
I tried to fix some problems with code style. Besides, I installed lint, set it with airbnb code style and tried to check the project and it showed a lot of problems. So I offer to discuss this issue not related to this particular pull request